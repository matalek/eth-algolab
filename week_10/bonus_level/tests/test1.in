10
6
82 13 91 84 12 97
92 22 63 31 9 55
28 19 55 100 96 100
97 97 15 73 98 99
96 11 49 80 80 29
14 0 42 11 92 64
6
80 88 96 50 66 80
3 36 85 21 94 68
68 40 76 74 75 47
39 42 66 17 17 30
71 80 3 31 27 88
4 15 9 100 83 83
6
70 12 32 77 95 49
3 67 44 12 38 21
77 5 80 3 18 41
49 46 45 49 65 80
71 93 76 81 27 71
68 0 66 71 16 65
6
12 46 50 78 96 57
34 88 59 81 22 1
75 82 25 82 51 94
70 41 89 42 96 58
55 15 14 76 15 23
26 81 84 99 25 33
6
82 30 24 1 93 21
35 91 19 85 25 96
62 78 47 99 35 6
83 80 59 60 55 74
92 70 28 68 76 39
76 56 38 21 57 53
6
7 40 5 35 53 59
78 35 94 97 13 15
57 39 47 39 1 73
34 39 16 93 80 44
31 87 53 62 16 12
60 47 26 34 66 53
6
69 72 75 99 45 72
8 92 23 51 92 56
15 50 83 46 54 55
100 45 7 86 44 61
10 50 97 98 0 3
78 98 82 36 87 68
6
8 34 40 86 26 4
80 66 43 75 91 13
18 99 26 9 14 28
13 81 87 7 58 63
55 0 14 68 86 53
62 44 35 20 51 13
6
40 38 7 77 24 4
12 25 18 50 24 83
42 99 5 83 91 30
95 4 49 25 49 54
34 89 90 34 37 3
11 16 78 88 39 21
6
24 27 40 49 9 22
13 49 95 96 96 65
58 76 6 43 23 55
35 5 82 32 1 40
4 91 17 81 65 26
73 13 65 49 45 38
