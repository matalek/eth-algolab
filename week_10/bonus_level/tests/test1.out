1464
1302
1185
1192
1279
1109
1287
1167
1080
1069
